### ISSUE TITLE
unable to add Ids

### ISSUE THREAD

**Issue by stoutouloutou** [hekmon/rtetempo#20]
Hello,
I installes rtetempo via HACS on my docker installation of homeassistant (official stable image), but when I validate the ids, I get an "Unknown error occurred"

the logs arror : "TypeError: HomeAssistant.__new__() missing 1 required positional argument: 'config_dir'"

Can someone help me finding where it comes from ?
Thank you !

The full logs shows : 

```
home-assistant   | 2023-09-08 12:06:18.909 ERROR (MainThread) [aiohttp.server] Error handling request
home-assistant   | Traceback (most recent call last):
home-assistant   |   File "/usr/local/lib/python3.11/site-packages/aiohttp/web_protocol.py", line 433, in _handle_request
home-assistant   |     resp = await request_handler(request)
home-assistant   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/local/lib/python3.11/site-packages/aiohttp/web_app.py", line 504, in _handle
home-assistant   |     resp = await handler(request)
home-assistant   |            ^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/local/lib/python3.11/site-packages/aiohttp/web_middlewares.py", line 117, in impl
home-assistant   |     return await handler(request)
home-assistant   |            ^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/src/homeassistant/homeassistant/components/http/security_filter.py", line 85, in security_filter_middleware
home-assistant   |     return await handler(request)
home-assistant   |            ^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/src/homeassistant/homeassistant/components/http/forwarded.py", line 100, in forwarded_middleware
home-assistant   |     return await handler(request)
home-assistant   |            ^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/src/homeassistant/homeassistant/components/http/request_context.py", line 28, in request_context_middleware
home-assistant   |     return await handler(request)
home-assistant   |            ^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 80, in ban_middleware
home-assistant   |     return await handler(request)
home-assistant   |            ^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 236, in auth_middleware
home-assistant   |     return await handler(request)
home-assistant   |            ^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/src/homeassistant/homeassistant/components/http/headers.py", line 31, in headers_middleware
home-assistant   |     response = await handler(request)
home-assistant   |                ^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 148, in handle
home-assistant   |     result = await handler(request, **request.match_info)
home-assistant   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/src/homeassistant/homeassistant/components/http/decorators.py", line 63, in with_admin
home-assistant   |     return await func(self, request, *args, **kwargs)
home-assistant   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/src/homeassistant/homeassistant/components/config/config_entries.py", line 177, in post
home-assistant   |     return await super().post(request, flow_id)
home-assistant   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/src/homeassistant/homeassistant/components/http/data_validator.py", line 72, in wrapper
home-assistant   |     result = await method(view, request, data, *args, **kwargs)
home-assistant   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/src/homeassistant/homeassistant/helpers/data_entry_flow.py", line 110, in post
home-assistant   |     result = await self._flow_mgr.async_configure(flow_id, data)
home-assistant   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 298, in async_configure
home-assistant   |     result = await self._async_handle_step(
home-assistant   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 394, in _async_handle_step
home-assistant   |     result: FlowResult = await getattr(flow, method)(user_input)
home-assistant   |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
home-assistant   |   File "/config/custom_components/rtetempo/config_flow.py", line 50, in async_step_user
home-assistant   |     hass = HomeAssistant()
home-assistant   |            ^^^^^^^^^^^^^^^
home-assistant   | TypeError: HomeAssistant.__new__() missing 1 required positional argument: 'config_dir'
```

**Comment by hekmon** [comment]
That's... weird.

I am guessing that the constructor has changed in 2023.9.0 and that all existing installations are fine but new one will fail.

I need to look into that when a bit a free time arise. In the meantime can you try this workaround ?

- Downgrade to the last HA 2023.8.X
- Install rtetempo
- Upgrade HA to 2023.9.0

If this is what I think it is, it should do the trick for now.

**Comment by stoutouloutou** [comment]
Hello.
Indeed, it worked !
Thank you.

**Comment by hekmon** [comment]
It should be fixed with [v1.3.2](https://github.com/hekmon/rtetempo/releases/tag/v1.3.2)
